src/library-authoring/studio-header/StudioHeader.jsx

```diff
@@ -1,13 +1,11 @@
-import React, { useContext } from 'react';
-import { Route, Link } from 'react-router-dom';
+import React from 'react';
+import { Link } from 'react-router-dom';
 import { connect } from 'react-redux';
-import { Dropdown, Icon } from '@edx/paragon';
 import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
-import { AppContext } from '@edx/frontend-platform/react';
 import { ensureConfig } from '@edx/frontend-platform/config';
+import { Header as StudioHeader, MenuContentItem } from '@edx/frontend-component-header';
```

Comment:
Looks like MenuContentItem [no longer exists](https://github.com/openedx/frontend-component-header/blob/master/src/index.jsx#L5-L7).  We'll need a different strategy, here.